core/src/main/java/ai/timefold/solver/core/impl/domain/solution/cloner/PlanningCloneable.java

```diff
@@ -0,0 +1,5 @@
+package ai.timefold.solver.core.impl.domain.solution.cloner;
+
+public interface PlanningCloneable<T> {
```

Comment:
The interface needs a good description of what it's for, what will happen when used, and how the method should be implemented.